airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -34,6 +35,146 @@
+                                "schemaless": {
+                                    "title": "Schemaless",
+                                    "default": False,
+                                    "type": "boolean"
+                                },
+                                "validation_policy": {
+                                    "title": "Validation Policy",
+                                    "type": "string"
+                                },
+                                "input_schema": {
+                                    "title": "Input Schema",
+                                    "type": "object"
+                                },
+                                "primary_key": {
+                                    "title": "Primary Key",
+                                    "anyOf": [
+                                        {
+                                            "type": "string"
+                                        },
+                                        {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "string"
+                                            }
+                                        },
+                                        {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "array",
+                                                "items": {
+                                                    "type": "string"
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                "days_to_sync_if_history_is_full": {
+                                    "title": "Days To Sync If History Is Full",
+                                    "default": 3,
+                                    "type": "integer"
+                                },
+                                "format": {
+                                    "anyOf": [
+                                        {
+                                            "title": "Format",
+                                            "type": "object",
+                                            "additionalProperties": {
+                                                "title": "CsvFormat",
+                                                "type": "object",
+                                                "properties": {
+                                                    "delimiter": {
+                                                        "title": "Delimiter",
+                                                        "default": ",",
+                                                        "type": "string"
```

Comment:
I think it's a missed opportunity that this spec does not inherit `description` and `title` fields from the base spec. Each connector should not have to re-explain what this field does when the base class can do a good job of it

Reply:
also in that direction, the titles in a lot of these fields are not easily understood by an end user or look like an abbreviation e.g it should be `Quote Character` instead of `Char` etc..